capi/VideoDecoderCapi.h

```diff
@@ -20,8 +20,8 @@
 #ifndef __ENABLE_CAPI__
 #define __ENABLE_CAPI__ 1
 #endif
-#include "interface/VideoDecoderDefs.h"
-#include "interface/VideoCommonDefs.h"
+#include "VideoDecoderDefs.h"
+#include "VideoCommonDefs.h"
```

Comment:
This is the place where needs to be changed only as this is the header on ${PREFIX}/include/libyami